babel.cjs.js

```diff
@@ -14,7 +14,9 @@ module.exports = {
   'plugins': [
     '@babel/plugin-transform-typescript',
     '@babel/plugin-proposal-class-properties',
-    '@babel/plugin-transform-runtime',
+    ['@babel/plugin-transform-runtime', {
+      corejs: 3
```

Comment:
Object.fromEntries is only available since Node v12, add `corejs: 3` here to bring in babel runtime function to compatible with Node v11.x.